src/Middleware/CORS/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,118 @@
+#nullable enable
+Microsoft.AspNetCore.Builder.CorsEndpointConventionBuilderExtensions
+Microsoft.AspNetCore.Builder.CorsMiddlewareExtensions
+Microsoft.AspNetCore.Cors.CorsPolicyMetadata
```

Comment:
How do these Unshipped and Shipped files work? I'd expect Unshipped to be the APIs new to 5.0 and Shipped to be the 3.1 baseline? Most of the APIs in this Unshipped file are not new.

Reply:
We manually move to the Shipped file once we ship.  Right now I'm just trying to get everything into the unshipped files as a baseline, and then we'll take our 5.0 surface area and move it to shipped at once.